fala/apps/adviser/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Feedback(models.Model):
+    satisfied = models.BooleanField(null=True, blank=True)
+    comment = models.TextField(null=True, blank=True)
```

Comment:
Null is not recommended on Char or Text fields (or their derivatives) in Django. Basically you don't want empty strings _and_ nulls in a column, and [Django's convention](https://docs.djangoproject.com/en/5.0/ref/models/fields/#null) is to disallow null and therefore use empty strings